h/static/scripts/annotator/plugin/toolbar.coffee

```diff
@@ -12,16 +12,22 @@ makeButton = (item) ->
 class Annotator.Plugin.Toolbar extends Annotator.Plugin
   PUSHED_CLASS = 'annotator-pushed'
 
+  touch = /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent)
+
   events:
-    '.annotator-toolbar mouseenter': 'show'
-    '.annotator-toolbar mouseleave': 'hide'
-    'setTool': 'onSetTool'
     'setVisibleHighlights': 'onSetVisibleHighlights'
 
-  html: '<div class="annotator-toolbar annotator-hide"></div>'
+  html: '<div class="annotator-toolbar"></div>'
 
   pluginInit: ->
     @annotator.toolbar = @toolbar = $(@html)
+    if touch
+      # When there is a selection on touch devices show/hide highlighter
+      document.addEventListener "selectstart", ->
+        if window.getSelection().toString() != ""
+          annotator.plugins.Toolbar.showHighlightButton(false)
```

Comment:
You're in `annotator.plugins.Toolbar`. Fat arrow bind this closure and you can just use `this`.